### PR TITLE
(EWL-1136): Display video information only on fullscreen mode

### PR DESCRIPTION
**Pull request by psaint-j**
# About the type of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Test plan

- [x] Empty data
- [x] Invalid data
- [x] Correct data

# Checklist
### Before code review

- [x] Move Jira ticket into PR
- [x] Check with another developers that features sounds good

### After code review

- [ ] Move Jira ticket into QA

### After QA validation
If conflicts are present in github:
  - [ ] `git pull --rebase origin master`
  - [ ] Fix conflicts
  - [ ] `git rebase --continue` repeat until there is no more conflicts
  - [ ] `git push --force-with-lease`

If there's no conflicts
- [ ] squash and merge
- [ ] delete branch

### CODE REVIEW

File: src/components/JWPlayerETP/PlayerSkin/index.js [hunk 1]
```diff
@@ -50,6 +50,7 @@ const PlayerSkin = ({
 
   const showSpinner = !active && isBuffering;
   const showControls = showSpinner === false && (!isPlaying || active);
+  const showTitle = isFullscreen ? title : '';
```
Comment (JulienKode) [inline src/components/JWPlayerETP/PlayerSkin/index.js]: Can we test that path ?

Comment (psaint-j) [reply src/components/JWPlayerETP/PlayerSkin/index.js]: Yes i will add some tests

File: src/components/JWPlayerETP/PlayerSkin/index.js [hunk 2]
```diff
@@ -78,7 +79,7 @@ const PlayerSkin = ({
               onSeek={onSeekKeepInteraction}
               onVolume={onVolumeKeepInteraction}
               onMute={onMuteKeepInteraction}
-              title={title}
+              title={showTitle}
```
Comment (MathieuDebit) [inline src/components/JWPlayerETP/PlayerSkin/index.js]: I think you can simplify it by only putting `isFullscreen && title` or boolean `isFullscreen ? title : ''`

Comment (psaint-j) [reply src/components/JWPlayerETP/PlayerSkin/index.js]: ok for the second idea

File: src/components/JWPlayerETP/PlayerSkin/index.test.js [hunk 3]
```diff
@@ -270,4 +271,38 @@ describe('Components|PlayerSkin', () => {
...
+
+  describe('Display title only on Fullscreen', () => {
+    let wrapper;
+    let onFullscreenChange;
+
+    beforeEach(() => {
+      onFullscreenChange = jest.fn();
+    });
+
+    it('Should display title when Fullscreen is activate', () => {
```
Comment (MathieuDebit) [inline src/components/JWPlayerETP/PlayerSkin/index.test.js]:
```suggestion
	    it('Should display title when Fullscreen is active', () => {
```

File: src/components/JWPlayerETP/PlayerSkin/index.test.js [hunk 4]
```diff
@@ -270,4 +271,38 @@ describe('Components|PlayerSkin', () => {
...
+      const props = createDefaultProps({ isPlaying: true });
+      wrapper = shallow(<PlayerSkin {...props} />);
+
+      // When
+      wrapper.find(Controls).prop('onFullscreenChange')();
+
+      // Expect
+      expect(wrapper.find(Controls).prop('title')).toEqual('title');
+    });
+    it('Should not display title when Fullscreen is deactivate', () => {
```
Comment (MathieuDebit) [inline src/components/JWPlayerETP/PlayerSkin/index.test.js]:
```suggestion
	  it('Should display title when Fullscreen is inactive', () => {
```

File: src/components/JWPlayerETP/PlayerSkin/index.test.js [hunk 5]
```diff
@@ -270,4 +271,38 @@ describe('Components|PlayerSkin', () => {
...
+      const props = createDefaultProps({ isPlaying: true });
+      wrapper = shallow(<PlayerSkin {...props} />);
+
+      // When
+      wrapper.find(Controls).prop('onFullscreenChange')();
+
+      // Expect
+      expect(wrapper.find(Controls).prop('title')).toEqual('title');
+    });
+    it('Should not display title when Fullscreen is inactive', () => {
```
Comment (MathieuDebit) [inline src/components/JWPlayerETP/PlayerSkin/index.test.js]:
```suggestion

    it('Should not display title when Fullscreen is inactive', () => {
```

**Pull request merged**